pkg/collector/parser/receiver_otlp_test.go

```diff
@@ -65,15 +65,14 @@ func TestOTLPExposeDefaultPorts(t *testing.T) {
 		seen       bool
 	}{
 		"otlp-grpc":        {portNumber: 4317},
-		"otlp-grpc-legacy": {portNumber: 55680},
```

Comment:
Looks like I can't update your PR. Please, run `make fmt` and update the PR.

Reply:
done. thanks!